release.sh

```diff
@@ -293,7 +293,6 @@ packages/fxa-content-server
 packages/fxa-customs-server
 packages/fxa-email-event-proxy
 packages/fxa-email-service
-packages/fxa-event-broker
```

Comment:
don't we want to include this as part of the release process or?

Reply:
We will, yes, think this was caught during a prior commit that was squashed.